app/containers/Bookings/Details/OffenderDetails/index.js

```diff
@@ -155,6 +104,32 @@ const OffenderDetails = ({ offenderDetails, showPhoto }) => {
         </div>
       </div>
 
+      <div className="row">
+        <div className="col-md-12">
+          <h3 className="heading-medium">Address Details</h3>
```

Comment:
`import { H3 } from '@govuk-react/heading'`  and then use `<H3></H3>`.  You can pass in `size="MEDIUM"` if it needs it.

Reply:
as discussed - there is some styling on this component which makes this a non-trivial fix as so out of scope for this PR.